src/docs/development/data-and-backend/json.md

```diff
@@ -127,13 +130,13 @@ with the JSON string as the method argument.
 
 <!-- skip -->
 ```dart
-Map<String, dynamic> user = json.decode(json);
+Map<String, dynamic> user = json.decode(user_data_str);
```

Comment:
@kwalrath @kevmoo - should we be promoting use of `jsonDecode()` here instead? /cc @Hixie @sfshaza2

Reply:
I personally like `jsonDecode()`, but not everyone agrees. @kevmoo?

Reply:
Ditto! Avoid confusion w/ local var/param named `json`

Reply:
FYI, I've created: _Code cleanup: `json.decode()` -> `jsonDecode()`_ #2021